LadybugTools_oM/Enum/EpwKeys.cs

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2024, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                     
+ *                                      
+ * The BHoM is free software: you can redistribute it and/or modify     
+ * it under the terms of the GNU Lesser General Public License as published by_
+ * the Free Software Foundation, either version 3.0 of the License, or    
+ * (at your option) any later version.                    
+ *                                      
+ * The BHoM is distributed in the hope that it will be useful,      
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of       
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the         
+ * GNU Lesser General Public License for more details.            
+ *                                      
+ * You should have received a copy of the GNU Lesser General Public License   
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.  _
+ */
+
+using System.ComponentModel;
+
+namespace BH.oM.LadybugTools
+{
+    [Description("An enum for keys that frequently appear in epw files.")]
+    public enum EpwKey
+    {
+        Undefined,
+        Aerosol_Optical_Depth,
```

Comment:
Don't use `_` in enum options.

Use the [`DisplayText`](https://github.com/BHoM/BHoM/blob/develop/BHoM/Attributes/DisplayTextAttribute.cs) attribute to give a human-readable name to the options.

Reply:
This is more about having a EPW-friendly name rather than a human-readable name. E.g. for line 31, the enum needs to map exactly to the string "Aerosol Optical Depth" in order to match a key in the EPW's dictionary. Will DisplayText work for this?

Reply:
`DisplayText` will allow you to have an enum option of `AerosolOpticalDepth` display in the UI as `Aerosol Optical Depth`.

If you need the `_` to map to keys in the EPW dictionary, I would recommend having a convert method in the adapter that is called on a pull/push to convert between a BHoM enum and the relevant key. That is slightly more work though so I would be willing to not hold up this PR for it if that helps?

Reply:
I think it's not too much work to make a convert method, as there is already some string manipulation happening on the python end to get these into a useful format (whos idea was it to use spaces in EPW dictionary keys?)